migration/migration.go

```diff
@@ -157,6 +157,8 @@ func getMigrations() migrations {
 	// version 26
 	m = append(m, steps{executeSQLFile("026-areas.sql")})
 
+	// Version 27
+	m = append(m, steps{executeSQLFile("027-iterations-parent-path-ltree.sql")})
```

Comment:
now, Updated to 29 :) https://github.com/almighty/almighty-core/pull/713/commits/7f455e9df4c95dfef39719ec06417d4a77e2f131